mobile-widgets/qml/main.qml

```diff
@@ -737,12 +735,12 @@ if you have network connectivity and want to sync your data to cloud storage."),
 
 	onWidthChanged: {
 		manager.appendTextToLog("Window width changed to " + width + " orientation " + Screen.primaryOrientation)
-		if (subsurfaceTheme.initialWidth !== undefined) {
-			if (width !== subsurfaceTheme.initialWidth && rootItem.firstChange) {
+		if (initialWidth !== undefined) {
+			if (width !== initialWidth && rootItem.firstChange) {
```

Comment:
so it is entirely possible that I am full of nonsense here, but IIRC the reason this was done in the horrible ugly way it was done was to ensure that things weren't set too early. There is this very odd thing that is happening on some devices where the width of the screen changes on us about half way through the initialization phase. And it took me several weeks of testing with all of my devices and with the help of several Subsurface-mobile users to come up with a sequence of steps that appeared to be completely brain-dead, but that lead to us having the correct width on all of the various phones (I seem to remember that this was mostly Android devices, but I'm no longer sure).

So while I think that on the surface this sounds logical, I am very worried about breaking something that took me so much time and effort to get right.

Reply:
I will drop that commit and force push a bit later

Reply:
However it is really weird that initialWidth is broken but initialHeight is not...especially because there no explanation.

Reply:
Really weird does not begin to cover it. This whole thing was driving me nuts.
IIRC I only track the width because they always changed together, and all I try to track with this is "when do we have the 'real' starting dimensions so that we can set the `gridUnit` and the `columnWidth` and all that".

It's entirely possible (even likely) that I'm just doing something fundamentally wrong here. But I'm sure you can see that I am rather nervous to risk having to go all of this again...